docs/core/diagnostics/performance-tracing.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Performance Tracing
+description: A guide to tracing in .NET
+ms.date: 05/19/2023
+---
+# Performance Tracing Overview
+
+Performance tracing is the ability to collect detailed diagnostic information about what is happening inside .NET processes, and includes telemetry information for the Runtime, GC, Libraries and application code.
+
+There are a number of aspects to performance tracing for .NET, including:
+
+## How events are sourced
+
+There are two main mechanisms within .NET for providing events for use with performance tracing:
```

Comment:
To the best of my knowledge DiagnosticSource is not often used for performance tracing. The main uses I've seen are APM tools and OpenTelemetry. Occasionally it gets used in combination with DiagnosticSourceEventSource for things like VS profiler database profiling.

Reply:
I think we need a better title than just tracing - Event Tracing or Performance & Diagnostics tracing are ideas I had.